app/components/TrackCard/index.js

```diff
@@ -0,0 +1,122 @@
+/* eslint-disable no-console */
+/**
+ *
+ * TrackCard
+ *
+ */
+
+import React, { useRef, useState } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+import { Card, Button } from 'antd';
+import { colors } from '@themes';
+import If from '@components/If';
+import { PlayCircleOutlined, PauseCircleOutlined } from '@ant-design/icons';
+
+const { Meta } = Card;
+
+const CustomCard = styled(Card)`
+  & > div {
+    display: flex;
+  }
+  && {
+    width: 22rem;
+    margin: 1rem 0;
+    height: ${(props) => (props.height ? props.height : 10)}rem;
+    background-color: ${colors.trackCardColor};
+  }
+`;
+
+const Text = styled(Meta)`
+  && {
+    margin: 0.3rem;
+    width: 15em;
+  }
+`;
+
+const Wrapper = styled.div`
+  && {
+    padding: 0 1em;
+  }
+`;
+
+const Image = styled.img`
+  && {
+    width: 5rem;
+    height: ${(props) => (props.height ? props.height : 6)}rem;
+  }
+`;
+const CustomButton = styled(Button)`
+  display: flex !important;
```

Comment:
Do we need !important! Can we avoid using it! Can you please remove it form other places as well!